ui/src/components/witness/WitnessBanner.js

```diff
@@ -15,7 +15,7 @@ const WitnessBanner = () => {
     <Flex
       w="100vw"
       bgColor="#FFFFFF95"
-      h="12vh"
+      h={["50px","12vh"]}
```

Comment:
same comment about px. please change to vh or em

Reply:
Noted